ocs_ci/utility/utils.py

```diff
@@ -33,6 +33,7 @@
 from semantic_version import Version
 from tempfile import NamedTemporaryFile, mkdtemp
 
+from ocs_ci.ocs.node import get_nodes
```

Comment:
This is causing you circular dependency issue as in node module we import from this module here:

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/node.py#L25

So you can resolve this by moving this import to line: 3871